ledger/participant-integration-api/src/main/scala/platform/store/CompletionFromTransaction.scala

```diff
@@ -97,6 +97,7 @@ private[platform] object CompletionFromTransaction {
       status = maybeStatus,
       transactionId = transactionId,
       applicationId = applicationId,
+      submissionId = maybeSubmissionId.getOrElse(""),
```

Comment:
Why is the `match` below (its intention was to avoid assuming protobuf default values) not enough? Also, how do the `AppendOnlyCompletionDeduplicationInfoIT*` tests pass?

Reply:
This one is a leftover, sorry